disdrodb/tests/test_check_configs.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pytest
+
+
+from disdrodb.l0.check_configs import (
+    check_bin_consistency,
+    check_cf_attributes,
+    check_raw_data_format,
+    check_l0b_encoding,
+    check_yaml_files_exists,
+    check_variable_consistency,
+)
+from disdrodb.l0.standards import available_sensor_name
+
+PATH_TEST_FOLDERS_FILES = os.path.join(os.path.dirname(os.path.realpath(__file__)), "pytest_files")
+
+
+@pytest.mark.parametrize("sensor_name", available_sensor_name())
+def test_check_yaml_files_exists(sensor_name):
+    check_yaml_files_exists(sensor_name)
+
+
+@pytest.mark.xfail

```

Comment:
If you use l0b_encoding.yml instead of variables.yml ... this test must not fail ! 
Update accordingly after the code changes